week5/README.md

```diff
@@ -0,0 +1,52 @@
+# Week 5 Homework
+
+## Part 1 - Short Answer Questions
+
+### a) _________ are used to customize the appearance and behavior of a View.
+- Modifiers
+
+### b) The basic building block of a SwiftUI interface is a ______________.
+- View
+
+### c) Each View in SwiftUI is a ___________ type.
+- Struct
+
+### d) List two ways you can find the modifiers of a View.
+1. Inspecting the code in Xcode
+2. Using Xcode's SwiftUI Inspector
+
+### e) True or False: A modifier changes a View.
+- True
+
+### f) Modifiers are performed and returned in __________ from the _______________ they are applied to.
+- Sequence; View
+
+### g) When the data driving a View changes, that View has to be ____________ to reflect the change.
+- Re-rendered
+
+### h) True or False: The @Binding establishes a two-way connection between Views.
+- True
```

Comment:
Also a tricky one. Binding creates a two-way connection between a property that stores data, and a view that displays and changes the data. So false would be more appropriate.